plugins/obs-webrtc/webrtc/src/output.rs

```diff
@@ -0,0 +1,385 @@
+                                                duration: packet.duration,
+                                                ..Default::default()
+                                            };
+                                            match (packet.typ, peer_connection.connection_state()) {
+                                                (_, RTCPeerConnectionState::Failed | RTCPeerConnectionState::Closed) => Err(WorkerResult::Error(OutputStreamError::NetworkError)),
+                                                (EncodedPacketType::Audio, _) => audio_track.write_sample(&sample).await.map_err(|e| WorkerResult::Error(OutputStreamError::WriteError(e))),
+                                                (EncodedPacketType::Video, _) => video_track.write_sample(&sample).await.map_err(|e| WorkerResult::Error(OutputStreamError::WriteError(e)))
+                                            }
+                                        },
+                                        Some(Message::Error(e)) => Err(WorkerResult::Error(e)),
+                                        Some(Message::Close) | None => Err(WorkerResult::Close),
+                                    }
+                                }
+                                _ = interval.tick() => {
+                                    let pc_stats = peer_connection.get_stats().await;
+
+                                    let stats = &mut stats.write().unwrap();
+                                    if let Some(StatsReportType::Transport(transport_stats)) = pc_stats.reports.get("ice_transport") {
+                                        stats.bytes_sent = transport_stats.bytes_sent as u64;
+                                    }
+                                    Ok(())
+                                }
+                            };
+
+                           if let Err(e) = result {
+                                break 'worker_loop e;
+                           }
+                        }
+                    }
+                );
+
+                match result {
+                    WorkerResult::Close => {}
+                    WorkerResult::Error(e) => {
+                        if let Some(callback) = &mut *error_callback.lock().unwrap() {
+                            error!("Worker encountered error: {e:?}");
+                            callback(e);
+                        }
+                    }
+                }
+
+                info!("Exiting worker thread");
+            }
+        });
+
+        *self.worker_handle.lock().unwrap() = Some(worker_handle);
+    }
+
+    pub async fn new() -> Result<Self> {
+        let video_track = Arc::new(TrackLocalStaticSample::new(
+            RTCRtpCodecCapability {
+                mime_type: MIME_TYPE_H264.to_owned(),
+                clock_rate: 90000,
+                sdp_fmtp_line: "profile-level-id=428014; max-fs=3600; max-mbps=108000; max-br=1400"
```

Comment:
I think this is supposed to be `max-br=14000`? I'm guessing the line came from this example (or its reference) https://www.mef.net/wp-content/uploads/2020/09/IMTC1015_22003_1.pdf

Reply:
FWIW its seems likely that this value isn't used.